tests/functional/modules/test_zos_apf_func.py

```diff
@@ -74,63 +43,45 @@
 del_expected = """/*BEGINAPFLIST*/
 /*ENDAPFLIST*/"""
 
-
-def run_shell_cmd(hosts, cmdStr):
-    results = hosts.all.shell(cmd=cmdStr)
-    pprint(vars(results))
-    for result in results.contacted.values():
-        out = result.get("stdout")
-    return out
-
-
-def persistds_create(hosts):
-    cmdStr = "mvstmp APFTEST.PRST"
-    prstds = run_shell_cmd(hosts, cmdStr)[:30]
-    cmdStr = "dtouch -tseq {0}".format(prstds)
-    run_shell_cmd(hosts, cmdStr)
-    return prstds
-
-
-def persistds_delele(hosts, ds):
-    cmdStr = "drm {0}".format(ds)
-    run_shell_cmd(hosts, cmdStr)
+def clean_test_env(hosts, test_info):
+    cmdStr = "drm {0}".format(test_info['library'])
+    hosts.all.shell(cmd=cmdStr)
+    if test_info.get('persistent'):
+        cmdStr = "drm {0}".format(test_info['persistent']['data_set_name'])
+        hosts.all.shell(cmd=cmdStr)
 
 
-def set_test_env(hosts, test_info):
-    # results = hosts.all.zos_data_set(name=ds, type="SEQ")
+def test_add_del(ansible_zos_module):
+    hosts = ansible_zos_module
+    test_info = dict(library="", state="present", force_dynamic=True)
     cmdStr = "mvstmp APFTEST"
-    ds = run_shell_cmd(hosts, cmdStr)[:25]
+    results = hosts.all.shell(cmd=cmdStr)
```

Comment:
I understand now that the call to mvstmp with APFTEST was so that the code could honor the static dictionaries. Feel free to resolve this if you don't have time to change things.